api/src/main/java/com/cloud/offering/ServiceOffering.java

```diff
@@ -86,7 +86,7 @@
     /**
      * @return Does this service plan support Volatile VM that is, discard VM's root disk and create a new one on reboot?
      */
-    boolean getVolatileVm();
+    boolean isVolatileVm();
```

Comment:
supportVolitileVMs() would be more pleasing on the human eye. Not that that should hamper this change but what is the objective? to adhere to another naming convention or to make better code for reading? and seriously both are valid motives under circumstances, but I am really wondering.

Reply:
The intention of my pull requests is to make code more readable and understandable, thus further making developers to understand the code more easily with method names without more context information. 
But about method naming things, developers have two different opinions as you mentioned:
(1) keep the same naming convention as similar methods.
(2) make code more readable.

Reply:
@BruceKuiLiu I am with 2 all the way, with the exception of the use of these methods in code generation tools. In this last method I would expect a name like isForVolatileVms() or the ealier sugested name. Of cause the accessor is named after the field implying that the field is not properly named either.
I am +0 on this one. Try to find more reviews.